### DISCUSSION TITLE
Do I understand well storage needed for plots ?

### DISCUSSION THREAD

**Discussion by cashmeoussidexx** [Chia-Network/chia-blockchain#1763] [Q&A]
Hi, I have gone through Chia plotting basics, but I'm not sure if I understood well plot size required when it comes to "size required" & "temporary size required".
So is it like, let's say I want to create k=32 plot (101.4GB, temporary space: 332 GB). So when plot is being created, I need 332 GB temporary space, but once its created (which takes hours) I will need only 101.4 GB to 'keep it'. Am I correct here ?

If yes, then is that good practice, to have for example 1.5 TB SSD drive dedicated only to create plots (for creating 3 X k=32 plots simultaneously, which will take around 1TB temporary space), and then once they are created on my SSD drive, move them to my HDD drive so that they can 'grow' there on much bigger HDD (like 50 TB) ?

**Comment by depado** [comment]
Yes you got everything right! This is how most of us are plotting at the moment, using an SSD as tmp drive and a bigger hard drive to store the final plots.

**Reply by cashmeoussidexx** [reply]
Thank you for your answer ! Does processor or RAM matter here or not ? For example if my windows server has 1,5 TB SSD + 140 TB HDD I can basically fill whole 140 TB HDD with k=32 plots and my processor or RAM doesn't really matter here ? 
Or in so many plots I need some super powerful processor and huge amount of RAM ?

**Reply by depado** [reply]
No, farming doesn't require much energy, RAM or CPU. I'm personally using a Raspberry4 to do that and it doesn't even come close to being overused (1.3 average load with 4 cores, 1GB of 8GB RAM used).
Plotting, on the other hand, requires some more serious RAM/CPU usage.
